WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsBaseTableViewController.swift

```diff
@@ -0,0 +1,49 @@
+import UIKit
+
+/// Base class for site stats table view controllers
+///
+
+class SiteStatsBaseTableViewController: UIViewController {
+
+    let refreshControl = UIRefreshControl()
+
+    var tableStyle: UITableView.Style = .grouped
```

Comment:
I spent some time (prob too much time) trying to get this to work with an initialiser but because these ViewController's are loaded from storyboard I don't see a good way at the moment.  Ideally these would no longer use storyboard in future.   Happy to spend more time on this if you'd like